logic/src/commonMain/kotlin/com/wire/kalium/logic/GlobalKaliumScope.kt

```diff
@@ -81,6 +87,11 @@ class GlobalKaliumScope(
     private val globalConfigRepository: GlobalConfigRepository
         get() =
             GlobalConfigDataSource(globalPreferences.value.globalAppConfigStorage)
+
+    val validateEmailUseCase: ValidateEmailUseCase get() = ValidateEmailUseCaseImpl()
+    val validateUserHandleUseCase: ValidateUserHandleUseCase get() = ValidateUserHandleUseCaseImpl()
+    val validatePasswordUseCase: ValidatePasswordUseCase get() = ValidatePasswordUseCaseImpl()
```

Comment:
These moved here because they need to be accessible before we've fetched the version?

Reply:
All off the validation use cases are not depended on server or version and need to be exposed to the app. So, it doesn't make sense to strict them to Authentication scope since the app need to validate email even before without the having to communicate with the backend to get its version